lib/Manager.php

```diff
@@ -357,6 +357,34 @@ public function getRoomsForActor(string $actorType, string $actorId, array $sess
 		return $rooms;
 	}
 
+	/**
+	 * @param string $userId
+	 * @return Room[]
+	 */
+	public function getLeftOneToOneRoomsForUser(string $userId): array {
+		$query = $this->db->getQueryBuilder();
+		$helper = new SelectHelper();
+		$helper->selectRoomsTable($query);
+		$query->from('talk_rooms', 'r')
+			->where($query->expr()->eq('r.type', $query->createNamedParameter(Room::ONE_TO_ONE_CALL)))
+			->andWhere($query->expr()->like('r.name', $query->createNamedParameter('%' . $this->db->escapeLikeParameter(json_encode($userId)) . '%')));
+
+		$result = $query->execute();
+		$rooms = [];
+		while ($row = $result->fetch()) {
+			if ($row['token'] === null) {
```

Comment:
still relevant today or copy-paste issue ?

Reply:
copy pasta and will remain until we clean it up in all methods with one go